middleware/errorHandler.mw.js

```diff
@@ -1,6 +1,7 @@
 const logger = require("./../middleware/loggers/error.logger");
 
 exports.errorHandler = (error, req, res, next) => {
+  console.log(error);
```

Comment:
should be removed, i assume it was there for debug

Reply:
you are right, will do